week-1-ts-ecosystem/w01e01-typescript-basics/src/types/index.d.ts

```diff
@@ -0,0 +1,8 @@
+type Address = { street: string; no: number };
+
+export type User = {
+  name: string;
+  age: number;
+  email: string;
+  address: Address;
+};
```

Comment:
Types jest tutaj ładnie przygotowane, choć osobiście,

odkryłbym to `User.d.ts` jako osobny plik a `types` nazwał `model`
Bo tutaj mieszka nasz modal danych.